src/modules/sdk/objections/axios.client.ts

```diff
@@ -39,7 +39,11 @@ export const makeAPICall = async (config: AxiosRequestConfig): Promise<AxiosResp
     logger.debug(`Calling ${config.method} ${config.url}`);
     const axiosResponse: AxiosResponse = await axios.request<any>(config);
     try {
-      logger.debug(`axios response from api call : ${JSON.stringify(axiosResponse, null, 2)}`);
+      const loggableAxiosResponse = {
```

Comment:
thought i'd improve the logging from Axios a bit. we're only really interested in the status and data. The issues of 'circular references' were mostly references in the root level of the response object and not in 'data'. So logging the data and status allows us to avoid most of the issues. Still have an issue when logging a download but not worried about that as the 'data' is just bytes. The file header info gets logged elsewhere.

Reply:
and added a test to make sure we are logging these values.